specification/storagecache/resource-manager/Microsoft.StorageCache/preview/2020-08-13-aml-preview/storagecache.json

```diff
@@ -0,0 +1,2623 @@
+      "properties": {
+        "state": {
+          "description": "List of amlFilesystem health states.",
+          "enum": [
+            "Unknown",
+            "Unavailable",
+            "Available",
+            "Degraded",
+            "Transitioning",
+            "HydratingNamespace",
+            "Maintenance"
+          ],
+          "x-ms-enum": {
+            "name": "amlFilesystemHealthStateType",
+            "modelAsString": true
+          },
+          "type": "string"
+        },
+        "statusDescription": {
+          "description": "Describes of state.",
+          "type": "string"
+        }
+      }
+    },
+    "amlFilesystemEncryptionSettings": {
+      "description": "amlFilesystem encryption settings.",
+      "properties": {
+        "keyEncryptionKey": {
+          "x-ms-secret": true,
+          "$ref": "#/definitions/KeyVaultKeyReference",
+          "description": "Specifies the location of the encryption key in Key Vault."
+        }
+      }
+    },
+    "amlFilesystemHSMSettings": {
+      "description": "amlFilesystem HSM settings.",
+      "properties": {
+        "container": {
+          "description": "Container used for the amlFilesystem.",
+          "$ref": "#/definitions/URLString"
+        },
+        "importPrefix" : {
+          "description": "Path/Prefix for blobs in above container.  Only blobs containing this prefix get pre-hydrated into the cluster namespace",
+          "readOnly": true,
+          "type" : "string"
+        },
+        "exportPrefix" : {
+          "description": "Path/Prefix in above blob container where data gets exported when an archive task runs",
+          "type" : "string"
+        }
+      }
+    },
+    "amlFilesystemNamespaceHydrationStatus" : {
+      "description" : "Information about the amlFilesystem namespace hydration status",
+      "type" : "object",
+      "readOnly" : true,
+      "properties": {
+        "state" : {
+          "description" : "The state of the namespace hydration",
+          "type" : "string",
+          "enum" : [
+            "NotStarted",
+            "NotConfigured",
+            "InProgress",
+            "Failed",
+            "Complete"
+          ],
+          "x-ms-enum": {
+            "name": "NamespaceHydrationStatusType",
+            "modelAsString": true
+          }
+        },
+        "progress" : {
+          "description" : "Progress information for the namespace hydration",
+          "type" : "object",
+          "properties" : {
+            "blobCount" : {
+              "description" : "The number of Blobs processed.",
+              "type": "integer"
+            },
+            "totalFileSizeTiB" : {
+              "description" : "Aggregate size of blobs ingested so far in TiB",
+              "type": "number",
+              "format": "float"
+            },
+            "namesPerSec" : {
+              "type" : "integer",
+              "description": "Rate of ingest accounting for the last 30 seconds"
```

Comment:
I blinked at this description a few times before I wrapped my head around it. How about "during the last 30 seconds" or "over the last 30 seconds"?

Reply:
Fixed in next update.